Turborepo hanging in CI

### Verify canary release

- [X] I verified that the issue exists in the latest Turborepo canary release.

### Link to code that reproduces this issue

https://github.com/jakubmazanec/apps/pull/21

### What package manager are you using / does the bug impact?

npm

### What operating system are you using?

Windows

### Which canary version will you have in your reproduction?

2.0.0-canary.2

### Describe the Bug

After updating from 1.10.3 to 1.13.3, Turborepo freezes on a `test` task during GitHub workflow run (which uses Linux; https://github.com/jakubmazanec/apps/actions/runs/9357742231/job/25758162977). I was unable to figure out why, nor any further details, since it is working locally on my Windows laptop. Also, I couldn't actually check the canary version, since it has breaking changes and I cannot modify my config.

### Expected Behavior

It works, like it is working on a version 1.10.3.

### To Reproduce

In other issues I see mentions that Turborepo currently randomly freezes on Linux, so maybe it's related?

### Additional context

_No response_

I'm sorry, I cannot modify the workflows again, but I have logs from last week ago; after that, I just used older version of Turborepo, after reading issues like #7382 or #5773, because the logs shows only that Turborepo waits for task, but doesn't say why. The task itself was codegen for EdgeDB, but then I replaced it with other commands (don't remember what these were), and sometimes it worked, sometimes didn't (but consistently for each specific variation of the task). I'm guessing the issue will be really weird and I'm not sure me debugging it will help. Anyway, here are the logs:

```
2024-05-25T18:03:01.1048899Z ##[group]Run npm run test -- --filter=dram -vv
2024-05-25T18:03:01.1049360Z [36;1mnpm run test -- --filter=dram -vv[0m
2024-05-25T18:03:01.1107107Z shell: /usr/bin/bash -e {0}
2024-05-25T18:03:01.1107405Z env:
2024-05-25T18:03:01.1107927Z   NPM_CONFIG_USERCONFIG: /home/runner/work/_temp/.npmrc
2024-05-25T18:03:01.1108365Z   NODE_AUTH_TOKEN: XXXXX-XXXXX-XXXXX-XXXXX
2024-05-25T18:03:01.1108732Z   EDGEDB_INSTANCE: jakubmazanec/whisky
2024-05-25T18:03:01.1109069Z   EDGEDB_BRANCH: whisky
2024-05-25T18:03:01.1111924Z   EDGEDB_SECRET_KEY: ***
2024-05-25T18:03:01.1112209Z ##[endgroup]
2024-05-25T18:03:01.2430682Z 
2024-05-25T18:03:01.2430902Z > test
2024-05-25T18:03:01.2432109Z > turbo run test --no-daemon --filter=dram -vv
2024-05-25T18:03:01.2432713Z 
2024-05-25T18:03:01.2756917Z 2024-05-25T18:03:01.275+0000 [DEBUG] turborepo_lib::shim: Global turbo version: 1.13.3
2024-05-25T18:03:01.2759678Z 2024-05-25T18:03:01.275+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/apps/apps/package.json
2024-05-25T18:03:01.2796203Z 2024-05-25T18:03:01.279+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/apps/package.json
2024-05-25T18:03:01.2799253Z 2024-05-25T18:03:01.279+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/package.json
2024-05-25T18:03:01.2801681Z 2024-05-25T18:03:01.279+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/package.json
2024-05-25T18:03:01.2802944Z 2024-05-25T18:03:01.279+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/package.json
2024-05-25T18:03:01.2804111Z 2024-05-25T18:03:01.279+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /package.json
2024-05-25T18:03:01.2805199Z 2024-05-25T18:03:01.279+0000 [DEBUG] turborepo_lib::shim: Repository Root: /home/runner/work/apps/apps
2024-05-25T18:03:01.2806633Z 2024-05-25T18:03:01.279+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/apps/apps/node_modules/turbo-linux-64/package.json
2024-05-25T18:03:01.2808192Z 2024-05-25T18:03:01.279+0000 [DEBUG] turborepo_lib::shim: Local turbo path: /home/runner/work/apps/apps/node_modules/turbo-linux-64/bin/turbo
2024-05-25T18:03:01.2809313Z 2024-05-25T18:03:01.279+0000 [DEBUG] turborepo_lib::shim: Local turbo version: 1.13.3
2024-05-25T18:03:01.2810256Z 2024-05-25T18:03:01.279+0000 [DEBUG] turborepo_lib::shim: Currently running turbo is local turbo.
2024-05-25T18:03:01.2840455Z 2024-05-25T18:03:01.283+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.2843186Z 2024-05-25T18:03:01.284+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/apps/apps/package.json
2024-05-25T18:03:01.2859560Z 2024-05-25T18:03:01.285+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/apps/apps/package.json
2024-05-25T18:03:01.2861172Z 2024-05-25T18:03:01.285+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.2869649Z 2024-05-25T18:03:01.285+0000 [DEBUG] turborepo_lib::run::builder: skipping turbod since --no-daemon was passed
2024-05-25T18:03:01.2871901Z 2024-05-25T18:03:01.285+0000 [DEBUG] turborepo_telemetry: Starting telemetry event queue flush (num_events=10)
2024-05-25T18:03:01.2873798Z 2024-05-25T18:03:01.286+0000 [DEBUG] turborepo_telemetry: Done telemetry event queue flush
2024-05-25T18:03:01.2876213Z 2024-05-25T18:03:01.286+0000 [DEBUG] turborepo_repository::discovery: discovering packages using caching strategy
2024-05-25T18:03:01.2878384Z 2024-05-25T18:03:01.286+0000 [DEBUG] turborepo_repository::discovery: discovering packages using primary strategy
2024-05-25T18:03:01.2879766Z 2024-05-25T18:03:01.286+0000 [DEBUG] turborepo_repository::discovery: discovering packages using local strategy
2024-05-25T18:03:01.2948529Z 2024-05-25T18:03:01.294+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/apps/apps/apps/somewhere/package.json
2024-05-25T18:03:01.2950283Z 2024-05-25T18:03:01.294+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/apps/apps/apps/dram/package.json
2024-05-25T18:03:01.2951844Z 2024-05-25T18:03:01.294+0000 [DEBUG] turborepo_repository::discovery: discovering packages using caching strategy
2024-05-25T18:03:01.2953011Z 2024-05-25T18:03:01.294+0000 [DEBUG] turborepo_repository::discovery: discovering packages using caching strategy
2024-05-25T18:03:01.3022893Z 2024-05-25T18:03:01.301+0000 [DEBUG] log: No cached session for DnsName(DnsName(DnsName("telemetry.vercel.com")))
2024-05-25T18:03:01.3024938Z 2024-05-25T18:03:01.302+0000 [DEBUG] log: Not resuming any session
2024-05-25T18:03:01.3133347Z 2024-05-25T18:03:01.312+0000 [DEBUG] log: Using ciphersuite TLS13_AES_128_GCM_SHA256
2024-05-25T18:03:01.3135119Z 2024-05-25T18:03:01.312+0000 [DEBUG] log: Not resuming
2024-05-25T18:03:01.3136891Z 2024-05-25T18:03:01.312+0000 [DEBUG] log: TLS1.3 encrypted extensions: [Protocols([6832])]
2024-05-25T18:03:01.3138801Z 2024-05-25T18:03:01.312+0000 [DEBUG] log: ALPN protocol is Some(b"h2")
2024-05-25T18:03:01.3140223Z 2024-05-25T18:03:01.313+0000 [DEBUG] log: Ticket saved
2024-05-25T18:03:01.3332391Z 2024-05-25T18:03:01.333+0000 [DEBUG] turborepo_repository::discovery: discovering packages using caching strategy
2024-05-25T18:03:01.3335158Z 2024-05-25T18:03:01.333+0000 [DEBUG] turborepo_lib::run::task_access: TASK ACCESS TRACE: unable to restore config from cache
2024-05-25T18:03:01.3354334Z • Packages in scope: dram
2024-05-25T18:03:01.3355402Z • Running test in 1 packages
2024-05-25T18:03:01.3356210Z • Remote caching disabled
2024-05-25T18:03:01.3358310Z 2024-05-25T18:03:01.335+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3361367Z 2024-05-25T18:03:01.335+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3364068Z 2024-05-25T18:03:01.335+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3366485Z 2024-05-25T18:03:01.335+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3368918Z 2024-05-25T18:03:01.335+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3371341Z 2024-05-25T18:03:01.336+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3373771Z 2024-05-25T18:03:01.336+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3376182Z 2024-05-25T18:03:01.336+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3378652Z 2024-05-25T18:03:01.336+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3381154Z 2024-05-25T18:03:01.336+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3383450Z 2024-05-25T18:03:01.337+0000 [DEBUG] turborepo_telemetry: Starting telemetry event queue flush (num_events=10)
2024-05-25T18:03:01.3385352Z 2024-05-25T18:03:01.337+0000 [DEBUG] turborepo_telemetry: Done telemetry event queue flush
2024-05-25T18:03:01.3454596Z 2024-05-25T18:03:01.345+0000 [DEBUG] turborepo_lib::run::global_hash: global hash env vars ["EDGEDB_BRANCH", "EDGEDB_INSTANCE", "EDGEDB_SECRET_KEY"]
2024-05-25T18:03:01.3574143Z 2024-05-25T18:03:01.357+0000 [DEBUG] turborepo_lib::run::global_hash: external deps hash: 45b40dcd82af3717
2024-05-25T18:03:01.3576102Z 2024-05-25T18:03:01.357+0000 [DEBUG] turborepo_lib::run: running visitor
2024-05-25T18:03:01.3578119Z 2024-05-25T18:03:01.357+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3579829Z 2024-05-25T18:03:01.357+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3581201Z 2024-05-25T18:03:01.357+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3582564Z 2024-05-25T18:03:01.357+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3583717Z 2024-05-25T18:03:01.357+0000 [DEBUG] turborepo_lib::task_hash: auto detected framework for dram
2024-05-25T18:03:01.3584739Z 2024-05-25T18:03:01.357+0000 [DEBUG] turborepo_lib::task_hash: framework: vite, env_prefix: ["VITE_*"]
2024-05-25T18:03:01.3590338Z 2024-05-25T18:03:01.358+0000 [DEBUG] turborepo_lib::task_hash: task hash env vars for dram:codegen
2024-05-25T18:03:01.3590981Z  vars: []
2024-05-25T18:03:01.3591809Z 2024-05-25T18:03:01.358+0000 [DEBUG] turborepo_lib::task_graph::visitor: task dram#codegen hash is 09839c4ba504b8c4
2024-05-25T18:03:01.3593125Z 2024-05-25T18:03:01.358+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3594503Z 2024-05-25T18:03:01.359+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-05-25T18:03:01.3604557Z 2024-05-25T18:03:01.360+0000 [DEBUG] turborepo_lib::process::child: waiting for task
2024-05-25T18:03:02.3585941Z 2024-05-25T18:03:02.358+0000 [DEBUG] turborepo_telemetry: Starting telemetry event queue flush (num_events=6)
2024-05-25T18:03:02.3587410Z 2024-05-25T18:03:02.358+0000 [DEBUG] turborepo_telemetry: Done telemetry event queue flush
2024-05-25T18:06:01.3169465Z 2024-05-25T18:06:01.316+0000 [DEBUG] log: Sending warning alert CloseNotify
2024-05-25T18:12:35.0089034Z ##[error]The operation was canceled.
```

```
2024-07-23T22:57:42.6095494Z ##[group]Run npm run test -- --filter=dram -vvv --env-mode=loose
2024-07-23T22:57:42.6096290Z [36;1mnpm run test -- --filter=dram -vvv --env-mode=loose[0m
2024-07-23T22:57:42.6135936Z shell: /usr/bin/bash -e {0}
2024-07-23T22:57:42.6136254Z env:
2024-07-23T22:57:42.6136470Z   TURBO_UI: 0
2024-07-23T22:57:42.6136734Z   EDGEDB_INSTANCE: jakubmazanec/whisky
2024-07-23T22:57:42.6137083Z   EDGEDB_BRANCH: turborepo
2024-07-23T22:57:42.6139904Z   EDGEDB_SECRET_KEY: ***
2024-07-23T22:57:42.6140201Z ##[endgroup]
2024-07-23T22:57:42.7202938Z 
2024-07-23T22:57:42.7203427Z > test
2024-07-23T22:57:42.7204671Z > turbo run test --no-daemon --filter=dram -vvv --env-mode=loose
2024-07-23T22:57:42.7205365Z 
2024-07-23T22:57:42.7530413Z 2024-07-23T22:57:42.752+0000 [DEBUG] turborepo_lib::shim: Global turbo version: 2.0.10-canary.0
2024-07-23T22:57:42.7532399Z 2024-07-23T22:57:42.752+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/apps/apps/package.json
2024-07-23T22:57:42.7566554Z 2024-07-23T22:57:42.756+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/apps/package.json
2024-07-23T22:57:42.7568970Z 2024-07-23T22:57:42.756+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/package.json
2024-07-23T22:57:42.7571221Z 2024-07-23T22:57:42.756+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/package.json
2024-07-23T22:57:42.7573377Z 2024-07-23T22:57:42.756+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/package.json
2024-07-23T22:57:42.7575223Z 2024-07-23T22:57:42.756+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /package.json
2024-07-23T22:57:42.7576330Z 2024-07-23T22:57:42.756+0000 [DEBUG] turborepo_lib::shim: Repository Root: /home/runner/work/apps/apps
2024-07-23T22:57:42.7577751Z 2024-07-23T22:57:42.756+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/apps/apps/node_modules/turbo-linux-64/package.json
2024-07-23T22:57:42.7579414Z 2024-07-23T22:57:42.756+0000 [DEBUG] turborepo_lib::shim::local_turbo_state: Local turbo path: /home/runner/work/apps/apps/node_modules/turbo-linux-64/bin/turbo
2024-07-23T22:57:42.7580749Z 2024-07-23T22:57:42.756+0000 [DEBUG] turborepo_lib::shim::local_turbo_state: Local turbo version: 2.0.10-canary.0
2024-07-23T22:57:42.7581803Z 2024-07-23T22:57:42.756+0000 [DEBUG] turborepo_lib::shim: Currently running turbo is local turbo.
2024-07-23T22:57:42.7582869Z 2024-07-23T22:57:42.757+0000 [TRACE] log: registering event source with poller: token=Token(1), interests=READABLE
2024-07-23T22:57:42.7607742Z 2024-07-23T22:57:42.760+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.7656614Z 2024-07-23T22:57:42.765+0000 [DEBUG] turborepo_lib::process: spawning children with pty: false
2024-07-23T22:57:42.7659010Z 2024-07-23T22:57:42.765+0000 [TRACE] turborepo_lib::run::builder: performing run on "linux-64"
2024-07-23T22:57:42.7661880Z 2024-07-23T22:57:42.765+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/apps/apps/package.json
2024-07-23T22:57:42.7664831Z 2024-07-23T22:57:42.765+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.7667116Z 2024-07-23T22:57:42.765+0000 [DEBUG] turborepo_lib::run::builder: skipping turbod since --no-daemon was passed
2024-07-23T22:57:42.7668911Z 2024-07-23T22:57:42.766+0000 [DEBUG] turborepo_repository::discovery: discovering packages using caching strategy
2024-07-23T22:57:42.7671017Z 2024-07-23T22:57:42.766+0000 [DEBUG] turborepo_repository::discovery: discovering packages using primary strategy
2024-07-23T22:57:42.7673020Z 2024-07-23T22:57:42.766+0000 [DEBUG] turborepo_repository::discovery: discovering packages using local strategy
2024-07-23T22:57:42.7675450Z 2024-07-23T22:57:42.766+0000 [DEBUG] turborepo_telemetry: Starting telemetry event queue flush (num_events=10)
2024-07-23T22:57:42.7677118Z 2024-07-23T22:57:42.766+0000 [DEBUG] turborepo_telemetry: Done telemetry event queue flush
2024-07-23T22:57:42.7694144Z 2024-07-23T22:57:42.769+0000 [DEBUG] globwalk: processing includes: ["apps/*/package.json", "packages/*/package.json"]
2024-07-23T22:57:42.7695441Z 2024-07-23T22:57:42.769+0000 [DEBUG] globwalk: processing excludes: ["**/node_modules/**"]
2024-07-23T22:57:42.7697767Z 2024-07-23T22:57:42.769+0000 [DEBUG] globwalk: processed includes: ["/home/runner/work/apps/apps/apps/*/package.json", "/home/runner/work/apps/apps/packages/*/package.json"]
2024-07-23T22:57:42.7699877Z 2024-07-23T22:57:42.769+0000 [DEBUG] globwalk: processed excludes: ["/home/runner/work/apps/apps/**/node_modules/**"]
2024-07-23T22:57:42.7708301Z 2024-07-23T22:57:42.770+0000 [TRACE] log: registering event source with poller: token=Token(140667599760640), interests=READABLE | WRITABLE
2024-07-23T22:57:42.7732599Z 2024-07-23T22:57:42.773+0000 [DEBUG] log: No cached session for DnsName(DnsName(DnsName("telemetry.vercel.com")))
2024-07-23T22:57:42.7736566Z 2024-07-23T22:57:42.773+0000 [DEBUG] log: Not resuming any session
2024-07-23T22:57:42.7737521Z 2024-07-23T22:57:42.773+0000 [TRACE] log: Sending ClientHello Message {
2024-07-23T22:57:42.7738330Z     version: TLSv1_0,
2024-07-23T22:57:42.7738850Z     payload: Handshake {
2024-07-23T22:57:42.7739458Z         parsed: HandshakeMessagePayload {
2024-07-23T22:57:42.7740123Z             typ: ClientHello,
2024-07-23T22:57:42.7740666Z             payload: ClientHello(
2024-07-23T22:57:42.7741233Z                 ClientHelloPayload {
2024-07-23T22:57:42.7741850Z                     client_version: TLSv1_2,
2024-07-23T22:57:42.7742857Z                     random: a0b1505dd42711491ee0a3ebd061584b30e86a7f1a195f08b6e3ef00ee7c4f3a,
2024-07-23T22:57:42.7744208Z                     session_id: 2b1ebe10701dd5d9e5771f1c536dd47cc8fa6ee11cb6197e1c73856403a5f83c,
2024-07-23T22:57:42.7745213Z                     cipher_suites: [
2024-07-23T22:57:42.7745891Z                         TLS13_AES_256_GCM_SHA384,
2024-07-23T22:57:42.7746607Z                         TLS13_AES_128_GCM_SHA256,
2024-07-23T22:57:42.7747347Z                         TLS13_CHACHA20_POLY1305_SHA256,
2024-07-23T22:57:42.7748134Z                         TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
2024-07-23T22:57:42.7748944Z                         TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
2024-07-23T22:57:42.7749794Z                         TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,
2024-07-23T22:57:42.7750627Z                         TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
2024-07-23T22:57:42.7751427Z                         TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
2024-07-23T22:57:42.7752277Z                         TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256,
2024-07-23T22:57:42.7753130Z                         TLS_EMPTY_RENEGOTIATION_INFO_SCSV,
2024-07-23T22:57:42.7753648Z                     ],
2024-07-23T22:57:42.7753969Z                     compression_methods: [
2024-07-23T22:57:42.7754449Z                         Null,
2024-07-23T22:57:42.7754740Z                     ],
2024-07-23T22:57:42.7755016Z                     extensions: [
2024-07-23T22:57:42.7755367Z                         SupportedVersions(
2024-07-23T22:57:42.7755695Z                             [
2024-07-23T22:57:42.7756002Z                                 TLSv1_3,
2024-07-23T22:57:42.7756349Z                                 TLSv1_2,
2024-07-23T22:57:42.7756667Z                             ],
2024-07-23T22:57:42.7756943Z                         ),
2024-07-23T22:57:42.7757420Z                         ECPointFormats(
2024-07-23T22:57:42.7757744Z                             [
2024-07-23T22:57:42.7758109Z                                 Uncompressed,
2024-07-23T22:57:42.7758453Z                             ],
2024-07-23T22:57:42.7758723Z                         ),
2024-07-23T22:57:42.7759009Z                         NamedGroups(
2024-07-23T22:57:42.7759570Z                             [
2024-07-23T22:57:42.7759892Z                                 X25519,
2024-07-23T22:57:42.7760767Z                                 secp256r1,
2024-07-23T22:57:42.7761147Z                                 secp384r1,
2024-07-23T22:57:42.7761486Z                             ],
2024-07-23T22:57:42.7761766Z                         ),
2024-07-23T22:57:42.7762086Z                         SignatureAlgorithms(
2024-07-23T22:57:42.7762434Z                             [
2024-07-23T22:57:42.7762775Z                                 ECDSA_NISTP384_SHA384,
2024-07-23T22:57:42.7763189Z                                 ECDSA_NISTP256_SHA256,
2024-07-23T22:57:42.7763565Z                                 ED25519,
2024-07-23T22:57:42.7763926Z                                 RSA_PSS_SHA512,
2024-07-23T22:57:42.7764307Z                                 RSA_PSS_SHA384,
2024-07-23T22:57:42.7764680Z                                 RSA_PSS_SHA256,
2024-07-23T22:57:42.7765061Z                                 RSA_PKCS1_SHA512,
2024-07-23T22:57:42.7765450Z                                 RSA_PKCS1_SHA384,
2024-07-23T22:57:42.7765825Z                                 RSA_PKCS1_SHA256,
2024-07-23T22:57:42.7766171Z                             ],
2024-07-23T22:57:42.7766449Z                         ),
2024-07-23T22:57:42.7766785Z                         ExtendedMasterSecretRequest,
2024-07-23T22:57:42.7767197Z                         CertificateStatusRequest(
2024-07-23T22:57:42.7767567Z                             OCSP(
2024-07-23T22:57:42.7767950Z                                 OCSPCertificateStatusRequest {
2024-07-23T22:57:42.7768393Z                                     responder_ids: [],
2024-07-23T22:57:42.7768789Z                                     extensions: ,
2024-07-23T22:57:42.7769143Z                                 },
2024-07-23T22:57:42.7769449Z                             ),
2024-07-23T22:57:42.7769724Z                         ),
2024-07-23T22:57:42.7770007Z                         ServerName(
2024-07-23T22:57:42.7770313Z                             [
2024-07-23T22:57:42.7770636Z                                 ServerName {
2024-07-23T22:57:42.7771036Z                                     typ: HostName,
2024-07-23T22:57:42.7771464Z                                     payload: HostName(
2024-07-23T22:57:42.7771854Z                                         (
2024-07-23T22:57:42.7772275Z                                             74656c656d657472792e76657263656c2e636f6d,
2024-07-23T22:57:42.7772724Z                                             DnsName(
2024-07-23T22:57:42.7773164Z                                                 "telemetry.vercel.com",
2024-07-23T22:57:42.7773580Z                                             ),
2024-07-23T22:57:42.7773940Z                                         ),
2024-07-23T22:57:42.7774275Z                                     ),
2024-07-23T22:57:42.7774599Z                                 },
2024-07-23T22:57:42.7774910Z                             ],
2024-07-23T22:57:42.7775191Z                         ),
2024-07-23T22:57:42.7775549Z                         SignedCertificateTimestampRequest,
2024-07-23T22:57:42.7775954Z                         KeyShare(
2024-07-23T22:57:42.7776245Z                             [
2024-07-23T22:57:42.7776579Z                                 KeyShareEntry {
2024-07-23T22:57:42.7776966Z                                     group: X25519,
2024-07-23T22:57:42.7777541Z                                     payload: 25a0637cd29cae2f5a2b37906e9eb677785023e467809ccf3bff133404d8b33e,
2024-07-23T22:57:42.7778063Z                                 },
2024-07-23T22:57:42.7778371Z                             ],
2024-07-23T22:57:42.7778649Z                         ),
2024-07-23T22:57:42.7778956Z                         PresharedKeyModes(
2024-07-23T22:57:42.7779290Z                             [
2024-07-23T22:57:42.7779604Z                                 PSK_DHE_KE,
2024-07-23T22:57:42.7779952Z                             ],
2024-07-23T22:57:42.7780226Z                         ),
2024-07-23T22:57:42.7780659Z                         Protocols(
2024-07-23T22:57:42.7781086Z                             [
2024-07-23T22:57:42.7781393Z                                 6832,
2024-07-23T22:57:42.7781754Z                                 687474702f312e31,
2024-07-23T22:57:42.7782105Z                             ],
2024-07-23T22:57:42.7782384Z                         ),
2024-07-23T22:57:42.7782677Z                         SessionTicket(
2024-07-23T22:57:42.7783012Z                             Request,
2024-07-23T22:57:42.7783308Z                         ),
2024-07-23T22:57:42.7783566Z                     ],
2024-07-23T22:57:42.7783819Z                 },
2024-07-23T22:57:42.7784052Z             ),
2024-07-23T22:57:42.7784276Z         },
2024-07-23T22:57:42.7787306Z         encoded: 010001040303a0b1505dd42711491ee0a3ebd061584b30e86a7f1a195f08b6e3ef00ee7c4f3a202b1ebe10701dd5d9e5771f1c536dd47cc8fa6ee11cb6197e1c73856403a5f83c0014130213011303c02cc02bcca9c030c02fcca800ff010000a7002b00050403040303000b00020100000a00080006001d00170018000d001400120503040308070806080508040601050104010017000000050005010000000000000019001700001474656c656d657472792e76657263656c2e636f6d00120000003300260024001d002025a0637cd29cae2f5a2b37906e9eb677785023e467809ccf3bff133404d8b33e002d000201010010000e000c02683208687474702f312e3100230000,
2024-07-23T22:57:42.7790212Z     },
2024-07-23T22:57:42.7790431Z }
2024-07-23T22:57:42.7791485Z 2024-07-23T22:57:42.775+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/apps/apps/apps/dram/package.json
2024-07-23T22:57:42.7793410Z 2024-07-23T22:57:42.775+0000 [DEBUG] turborepo_repository::package_json: loading package.json from /home/runner/work/apps/apps/apps/somewhere/package.json
2024-07-23T22:57:42.7794781Z 2024-07-23T22:57:42.776+0000 [DEBUG] turborepo_repository::discovery: discovering packages using caching strategy
2024-07-23T22:57:42.7796827Z 2024-07-23T22:57:42.776+0000 [DEBUG] turborepo_repository::discovery: discovering packages using caching strategy
2024-07-23T22:57:42.7798533Z 2024-07-23T22:57:42.779+0000 [TRACE] log: We got ServerHello ServerHelloPayload {
2024-07-23T22:57:42.7799525Z     legacy_version: TLSv1_2,
2024-07-23T22:57:42.7800683Z     random: 28259505089fa7462de3aab6d99215f20c176f387ea5709ca3844b081e842572,
2024-07-23T22:57:42.7801931Z     session_id: 2b1ebe10701dd5d9e5771f1c536dd47cc8fa6ee11cb6197e1c73856403a5f83c,
2024-07-23T22:57:42.7802939Z     cipher_suite: TLS13_AES_128_GCM_SHA256,
2024-07-23T22:57:42.7803563Z     compression_method: Null,
2024-07-23T22:57:42.7804070Z     extensions: [
2024-07-23T22:57:42.7804517Z         SupportedVersions(
2024-07-23T22:57:42.7805021Z             TLSv1_3,
2024-07-23T22:57:42.7805451Z         ),
2024-07-23T22:57:42.7805832Z         KeyShare(
2024-07-23T22:57:42.7806257Z             KeyShareEntry {
2024-07-23T22:57:42.7806790Z                 group: X25519,
2024-07-23T22:57:42.7807709Z                 payload: b3109a12098e020ec804a3a46451ce62809ac07f06e8b8316f002620012a9e03,
2024-07-23T22:57:42.7808589Z             },
2024-07-23T22:57:42.7808970Z         ),
2024-07-23T22:57:42.7809324Z     ],
2024-07-23T22:57:42.7809672Z }
2024-07-23T22:57:42.7810689Z 2024-07-23T22:57:42.779+0000 [DEBUG] log: Using ciphersuite TLS13_AES_128_GCM_SHA256
2024-07-23T22:57:42.7811873Z 2024-07-23T22:57:42.779+0000 [DEBUG] log: Not resuming
2024-07-23T22:57:42.7812931Z 2024-07-23T22:57:42.779+0000 [TRACE] log: EarlyData rejected
2024-07-23T22:57:42.7813986Z 2024-07-23T22:57:42.779+0000 [TRACE] log: Dropping CCS
2024-07-23T22:57:42.7815355Z 2024-07-23T22:57:42.779+0000 [DEBUG] log: TLS1.3 encrypted extensions: [Protocols([6832])]
2024-07-23T22:57:42.7816831Z 2024-07-23T22:57:42.779+0000 [DEBUG] log: ALPN protocol is Some(b"h2")
2024-07-23T22:57:42.7896717Z 2024-07-23T22:57:42.779+0000 [TRACE] log: Server cert is [Certificate(b"0\x82\x04\xf30\x82\x03\xdb\xa0\x03\x02\x01\x02\x02\x12\x04e\x13y\x0b\xc4:a\x10\x91.\x1d\xee\xbb\x98\x9d\xaa\x8a0\r\x06\t*\x86H\x86\xf7\r\x01\x01\x0b\x05\0031\x0b0\t\x06\x03U\x04\x06\x13\x02US1\x160\x14\x06\x03U\x04\n\x13\rLet's Encrypt1\x0c0\n\x06\x03U\x04\x03\x13\x03R110\x1e\x17\r240703074859Z\x17\r241001074858Z0\x171\x150\x13\x06\x03U\x04\x03\x0c\x0c*.vercel.com0\x82\x01\"0\r\x06\t*\x86H\x86\xf7\r\x01\x01\x01\x05\0\x03\x82\x01\x0f\00\x82\x01\n\x02\x82\x01\x01\0\xedO\x0e\\-\xeeX\x05\x9d\xb3;\x93H9\x0fa\xa0m\\0$\xb3\xb1\xe9\xde\xe1|\xb00()\xa7H\x8e7\xe0\x89l\xe6^T\xd5\xd8$\x1d\xd0\xb3^\x84\x84\xfa\x9aE \xf7\xf6Y\x94\xa2 \xf8\xdbG\x9bNO\xb7\xb0B\xb6\x83\x0c\xac\xa9\xa2\xbc\x1f\xd7\x85|4\xd2n\xda'\x8e\xd3\x80\x88\xc9\xd0\xd8\xcct\x9b\x96\x1b\xec&\xf9\x90\xeb\x07/\x85>\xd1\xa8\x92\xf4\"\xdewN|*\xdc\x89\xa0\xd8K\x9d\x0e_+dGg~JT\xe3\x997\xee$\xd3\xfc\xaa\xe1\xb4\xb6\x01\xa6\"U?\xd7/h{*&\xa2\x8b#R:|\x1e\x108\xf7\xb7\x15\xe1\xb6\x11T\x07\xb2_\xb4\xa7\xd6\x98k\xc7H\xfa\t\xfb\x19O\xfa,\x1b\xb9\xf5\x0e\xc5\xaej\xc9\t\xa8\xaf8C\xa3f\xd4x\xb5O\x91|\xda\xa4\xad\xab\xd1\xd9K\x96\xd3\xbc\xe7\xe7\xe5\xfc@x\xa5}\xbd<\x82o\xb4%\x80\x06\xafb\xa6\xc6A\xf2\xd2\xa6F\xd6'\xb9?6\x1b\x17\xc4\x08\xed}q?\x95\x02\x03\x01\0\x01\xa3\x82\x02\x1b0\x82\x02\x170\x0e\x06\x03U\x1d\x0f\x01\x01\xff\x04\x04\x03\x02\x05\xa00\x1d\x06\x03U\x1d%\x04\x160\x14\x06\x08+\x06\x01\x05\x05\x07\x03\x01\x06\x08+\x06\x01\x05\x05\x07\x03\x020\x0c\x06\x03U\x1d\x13\x01\x01\xff\x04\x020\00\x1d\x06\x03U\x1d\x0e\x04\x16\x04\x14\x06\x13T\xb3qowY\x8a\x89\xfa-\x1eC\x93\xa3?\xc6Y\xec0\x1f\x06\x03U\x1d#\x04\x180\x16\x80\x14\xc5\xcfF\xa4\xea\xf4\xc3\xc0zl\x95\xc4-\xb0^\x92/&\xe3\xb90W\x06\x08+\x06\x01\x05\x05\x07\x01\x01\x04K0I0\"\x06\x08+\x06\x01\x05\x05\x070\x01\x86\x16http://r11.o.lencr.org0#\x06\x08+\x06\x01\x05\x05\x070\x02\x86\x17http://r11.i.lencr.org/0#\x06\x03U\x1d\x11\x04\x1c0\x1a\x82\x0c*.vercel.com\x82\nvercel.com0\x13\x06\x03U\x1d \x04\x0c0\n0\x08\x06\x06g\x81\x0c\x01\x02\x010\x82\x01\x03\x06\n+\x06\x01\x04\x01\xd6y\x02\x04\x02\x04\x81\xf4\x04\x81\xf1\0\xef\0v\0?\x17KO\xd7\"GX\x94\x1de\x1c\x84\xbe\r\x12\xed\x907\x7f\x1f\x85j\xeb\xc1\xbf(\x85\xec\xf8dn\0\0\x01\x90w\xc8bF\0\0\x04\x03\0G0E\x02 |\x83\n\xc2/\xbank\xce\x08\xc07\xf3Y\xf7\x12\xe2\xcb\xb4\x03Y\x14\x07\x18\x93\x8d\x90\x87u\xecO\xa8\x02!\0\xe7&^\x87\x83z\xd8\xc8\x11\xdaz\xa6\xda\xe7)\x89\xf6\x1d\xcd\xf5\xe1B\xcf\x10\\\xbd5\xb2\xab\x8d\x8aA\0u\0\x19\x98\x10q\t\xf0\xd6R.0\x80\xd2\x9e?d\xbb\x83n(\xcc\xf9\x0fR\x8e\xee\xdf\xceJ?\x16\xb4\xca\0\0\x01\x90w\xc8b\xed\0\0\x04\x03\0F0D\x02 +\xbf\xd0\x1f\x82qC\t\x10q\x1d\x8a\xcd\xec\xd3\xcd\r\xbd9D|qn\xc1\x1e\xd2\xcf&\xe8\xd4\x97\xfa\x02 {w\xcdg\xe4\x9a\xef\xf7*W\x1e)\x95#I\xf1\xd8h\x96\xb7\xda\xe8\x91d\x12\x1f\xdb\xd9AP)T0\r\x06\t*\x86H\x86\xf7\r\x01\x01\x0b\x05\0\x03\x82\x01\x01\0\x88Y=lj\x97\x7fa\xfb\x87\xa7\x80\x0eb\x0c\xd3\xb0\xf2\xed\xe4)L\xe8\xad\xf7\xcc5\xc0\xb4DU\xef^\x83\x05\xc0\xed\x01\xab\xa1}\xd5\xc9\x19\xc2a\xf3\xa1O&\xeb\xae\x8a\xacW\xb3R\xebMx\xb1\x84.\x8c\xe12Vm!\xc5:\xb1\xfa\xb2\xf7\xd7el\xa5/S2\xcb\xe7*\xa8;\xf7\xe3 \xdf,\n\xf8Z>\x99O\xa8\xd8\x7f\x95\x89\x0e\x8b~h:U\xa9o\x12pG\xcf\x12D9O\xfe\xfbg\xb1\xd4e\xbb\x1d\xea\xee\x13\xf1Un{\xbe\xe4\x15\xd2G\xe47\xadP\xb3\xfe\xe9L3ym\x92y8\xcb\x12\xb9\xb6A)@<0\xd5\x01\x9c\xc9\x1b\x867\x16\xb3\x95]\x0e\x13\x0b\xd6\x84\xa0]\no\x85\x18M\xed\xdf\x06\x1fM\x0b\xefj\xea\xf7e\xa0\n\xed\xd6\x0e\xbe\xc5\xc6\x17\x92\xc7z\xc5*\xf3\xd2\xb6);\xa2\xb2*\xcb\x10n\xa1\x07\xd0\x18\xd0v\x9cC\x9b\xc4\xd3\x1a\xd9S'\xd4\xf9q\xe4\xb8\xb85.l`\xb4hYi&\xe2B\xfdU\xb7"), Certificate(b"0\x82\x05\x060\x82\x02\xee\xa0\x03\x02\x01\x02\x02\x11\0\x8a}>\x13\xd6/0\xef#\x86\xbd)\x07k4\xf80\r\x06\t*\x86H\x86\xf7\r\x01\x01\x0b\x05\00O1\x0b0\t\x06\x03U\x04\x06\x13\x02US1)0'\x06\x03U\x04\n\x13 Internet Security Research Group1\x150\x13\x06\x03U\x04\x03\x13\x0cISRG Root X10\x1e\x17\r240313000000Z\x17\r270312235959Z031\x0b0\t\x06\x03U\x04\x06\x13\x02US1\x160\x14\x06\x03U\x04\n\x13\rLet's Encrypt1\x0c0\n\x06\x03U\x04\x03\x13\x03R110\x82\x01\"0\r\x06\t*\x86H\x86\xf7\r\x01\x01\x01\x05\0\x03\x82\x01\x0f\00\x82\x01\n\x02\x82\x01\x01\0\xba\x87\xbc\\\x1b\09\xcb\xca\n\xcd\xd4g\x10\xf9\x01<\xa5N\xa5a\xcb&\xcaR\xfb\x15\x01\xb7\xb9(\xf5(\x1e\xed'\xb3$\x189g\t\x0c\x08\xec\xe0:\xb0;w\x0e\xbd\xf3\xe59TA\x0cN\xaeA\xd6\x99t\xdeQ\xdb\xef{\xffX\xbd\xa8\xb7\x13\xf6\xde1\xd5\xf2r\xc9rj\x0b\x83t\x95\x9cF\0d\x14\x99\xf3\xb1\xd9\"\xd9\xcd\xa8\x92\xaa\x1c&z?\xfe\xefX\x05{\x08\x95\x81\xdbq\x0f\x8e\xfb\xe31\t\xbb\t\xbePM_\x8f\x91v=Z\x9d\x9e\x83\xf2\xe9\xc4f\xb3\xe1\x06fCH\x18\x80e\xa07\x18\x9a\x9b\x842\x97\xb1\xb2\xbd\xc4\xf8\x15\0\x9d'\x88\xfb\xe2c\x17\x96l\x9b'gK\xc4\xdb(^i\xc2y\xf0I\\\xe0$P\xe1\xc4\xbc\xa1\x05\xac{@m\0\xb4\xc2A?\xa7X\xb8/\xc5\\\x9b\xa5\xbb\t\x9e\xf1\xfe\xeb\xb0\x859\xfd\xa8\n\xefE\xc4x\xebe*\xc2\xcf_<\xde\xe3\\M\x1b\xf7\x0b'+\xaa\x0bBwSOyj\x1d\x87\xd9\x02\x03\x01\0\x01\xa3\x81\xf80\x81\xf50\x0e\x06\x03U\x1d\x0f\x01\x01\xff\x04\x04\x03\x02\x01\x860\x1d\x06\x03U\x1d%\x04\x160\x14\x06\x08+\x06\x01\x05\x05\x07\x03\x02\x06\x08+\x06\x01\x05\x05\x07\x03\x010\x12\x06\x03U\x1d\x13\x01\x01\xff\x04\x080\x06\x01\x01\xff\x02\x01\00\x1d\x06\x03U\x1d\x0e\x04\x16\x04\x14\xc5\xcfF\xa4\xea\xf4\xc3\xc0zl\x95\xc4-\xb0^\x92/&\xe3\xb90\x1f\x06\x03U\x1d#\x04\x180\x16\x80\x14y\xb4Y\xe6{\xb6\xe5\xe4\x01s\x80\x08\x88\xc8\x1aX\xf6\xe9\x9bn02\x06\x08+\x06\x01\x05\x05\x07\x01\x01\x04&0$0\"\x06\x08+\x06\x01\x05\x05\x070\x02\x86\x16http://x1.i.lencr.org/0\x13\x06\x03U\x1d \x04\x0c0\n0\x08\x06\x06g\x81\x0c\x01\x02\x010'\x06\x03U\x1d\x1f\x04 0\x1e0\x1c\xa0\x1a\xa0\x18\x86\x16http://x1.c.lencr.org/0\r\x06\t*\x86H\x86\xf7\r\x01\x01\x0b\x05\0\x03\x82\x02\x01\0N\xe2\x89]\n\x03\x1c\x908\xd0\xf5\x1f\xf9q\\\xf8\xc3\x8f\xb27\x88zo\xb0%\x1f\xed\xbe\xb7\xd8\x86\x06\x8e\xe9\t\x84\xcdr\xbf\x81\xf3\xfc\xca\xcfSH\xed\xbd\xf6iB\xd4\xa5\x11>5\xc8\x13\xb2\x92\x1d\x05_\xea.\xd4\xd8\xf8I\xc3\xad\xf5\x99\x96\x9c\xef&\xd8\xe1\xb4$\x0bH M\xfc\xd3T\xb4\xa9\xc6!\xc8\xe16\x1b\xffwd)\x17\xb9\xf0K\xef]\xea\xcdy\xd0\xbf\x90\xbf\xbe#\xb2\x90\xdaJ\xa9H1t\xa9D\x0b\xe1\xe2\xf6-\x83q\xa4u{\xd2\x94\xc1\x05\x19F\x1c\xb9\x8f\xf3\xc4tH%*\r\xe5\xf5\xdbC\xe2\xdb\x93\x9b\xb9\x19\xb4\x1f/\xdfj\x0e\x8f1\xd3c\x0f\xbb)\xdc\xddf,?\xb0\x1bgQ\xf8A<\xe4M\xb9\xac\xb8\xa4\x9cfc\xf5\xab\x85#\x1d\xccS\xb6\xabq\xae\xdc\xc5\x01q\xda6\xee\n\x18*2\xfd\t1|\x8f\xf6s\xe7\x9c\x9c\xb5J\x15jw\x82Z\xcf\xda\x8dE\xfe\x1f*d\x050>s\xc2\xc6\x0c\xb9\xd6;cJ\xabF\x03\xfe\x99\xc0F@'`c\xdfP:\x07G\xd8\x15J\x9f\xeaG\x1f\x99Z\x08b\x0c\xb6l3\x08M\xd78\xedH-.\x05h\xae\x80]\xefL\xdc\xd8 A_h\xf1\xbbZ\xcd\xe3\x0e\xb0\x0c1\x87\x9bC\xdeIC\xe1\xc8\x04?\xd1<\x1b\x87E0i\xa8\xa9r\x0ey\x12\x1c1\xd8>#W\xdd\xa7O\xa0\xf0\x1c\x81\xd1w\x1fo\xd6\xd2\xb9\xa8\xb3\x03\x16\x819K\x9fU\xae\xd2j\xe4\xb3\xbf\xea\xa5\xd5\x9fK\xa3\xc9\xd6;r\xf3J\xf6T\xab\x0c\xfc8\xf7`\x80\xdfn5\xcau\xa1T\xe4/\xbcn\x17\xc9\x1a\xa57\xb5\xa2\x9a\xba\xec\xf4\xc0uFOw\xa8\xe8YV\x91f-n\xde)\x81\xd6\xa6\x97\x05^dE\xbe,\xce\xeadBD\xb0\xc3O\xad\xf0\xb4\xdc\x03\xca\x99\x9b\t\x82\x95\x82\rc\x8af\xf9\x19r\xf8\xd5\xb9\x89\x10\xe2\x89\x98\t5\xf9\xa2\x1c\xbe\x92s#t\xe9\x9d\x1f\xd7;J\x9a\x84X\x10\xc2\xf3\xa7\xe25\xec~;E\xce0FRk\xc0\xc0")]
2024-07-23T22:57:42.7970935Z 2024-07-23T22:57:42.779+0000 [TRACE] log: Unvalidated OCSP response: [48, 130, 1, 244, 10, 1, 0, 160, 130, 1, 237, 48, 130, 1, 233, 6, 9, 43, 6, 1, 5, 5, 7, 48, 1, 1, 4, 130, 1, 218, 48, 130, 1, 214, 48, 129, 191, 161, 53, 48, 51, 49, 11, 48, 9, 6, 3, 85, 4, 6, 19, 2, 85, 83, 49, 22, 48, 20, 6, 3, 85, 4, 10, 19, 13, 76, 101, 116, 39, 115, 32, 69, 110, 99, 114, 121, 112, 116, 49, 12, 48, 10, 6, 3, 85, 4, 3, 19, 3, 82, 49, 49, 24, 15, 50, 48, 50, 52, 48, 55, 50, 51, 48, 57, 52, 55, 48, 48, 90, 48, 117, 48, 115, 48, 75, 48, 9, 6, 5, 43, 14, 3, 2, 26, 5, 0, 4, 20, 26, 82, 185, 180, 89, 228, 195, 51, 152, 33, 126, 134, 109, 75, 189, 138, 59, 214, 103, 206, 4, 20, 8, 185, 17, 59, 165, 208, 133, 24, 180, 234, 15, 160, 173, 159, 134, 30, 142, 252, 56, 39, 2, 18, 4, 101, 19, 121, 11, 196, 58, 97, 16, 145, 46, 29, 238, 187, 152, 157, 170, 138, 128, 0, 24, 15, 50, 48, 50, 52, 48, 55, 50, 51, 48, 57, 52, 55, 48, 48, 90, 160, 17, 24, 15, 50, 48, 50, 52, 48, 55, 51, 48, 48, 57, 52, 54, 53, 56, 90, 48, 13, 6, 9, 42, 134, 72, 134, 247, 13, 1, 1, 11, 5, 0, 3, 130, 1, 1, 0, 65, 186, 230, 108, 104, 132, 163, 24, 121, 5, 2, 158, 12, 187, 50, 28, 77, 17, 51, 179, 93, 105, 51, 82, 82, 206, 62, 0, 222, 211, 84, 174, 59, 220, 237, 18, 190, 35, 36, 35, 70, 86, 86, 190, 68, 128, 65, 221, 229, 118, 64, 33, 239, 226, 161, 113, 56, 252, 233, 76, 20, 186, 12, 5, 210, 197, 87, 96, 170, 78, 57, 218, 130, 190, 19, 42, 227, 39, 108, 69, 239, 51, 2, 103, 0, 92, 237, 80, 139, 42, 155, 13, 44, 44, 190, 143, 1, 94, 91, 207, 227, 61, 188, 1, 211, 28, 167, 168, 246, 201, 90, 121, 183, 209, 108, 87, 227, 88, 82, 246, 73, 131, 157, 88, 179, 204, 203, 220, 68, 90, 81, 134, 13, 222, 98, 249, 101, 130, 64, 184, 192, 62, 170, 192, 250, 204, 57, 93, 21, 226, 17, 36, 161, 105, 158, 144, 241, 249, 53, 147, 51, 52, 124, 38, 201, 61, 34, 124, 240, 31, 231, 213, 110, 30, 142, 161, 175, 13, 65, 22, 241, 127, 174, 82, 33, 164, 99, 99, 181, 58, 237, 134, 137, 30, 81, 106, 129, 34, 181, 98, 189, 215, 35, 236, 106, 215, 86, 206, 241, 11, 201, 181, 90, 172, 250, 235, 71, 187, 231, 242, 215, 158, 45, 53, 32, 215, 15, 86, 225, 24, 189, 166, 67, 115, 124, 242, 43, 182, 241, 152, 169, 104, 222, 197, 151, 176, 243, 89, 22, 215, 173, 32, 109, 177, 200, 185]
2024-07-23T22:57:42.7982466Z 2024-07-23T22:57:42.780+0000 [DEBUG] log: Ticket saved
2024-07-23T22:57:42.7983468Z 2024-07-23T22:57:42.780+0000 [TRACE] log: signal: Want
2024-07-23T22:57:42.7984665Z 2024-07-23T22:57:42.780+0000 [TRACE] log: signal found waiting giver, notifying
2024-07-23T22:57:42.7985977Z 2024-07-23T22:57:42.780+0000 [TRACE] log: poll_want: taker wants!
2024-07-23T22:57:42.7987048Z 2024-07-23T22:57:42.780+0000 [TRACE] log: signal: Want
2024-07-23T22:57:42.7987989Z 2024-07-23T22:57:42.780+0000 [TRACE] log: signal: Want
2024-07-23T22:57:42.7989479Z 2024-07-23T22:57:42.783+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:42.7991208Z 2024-07-23T22:57:42.783+0000 [TRACE] tokio_util::codec::framed_impl: frame decoded from buffer
2024-07-23T22:57:42.7992927Z 2024-07-23T22:57:42.783+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:42.7994621Z 2024-07-23T22:57:42.783+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:42.7996292Z 2024-07-23T22:57:42.783+0000 [TRACE] tokio_util::codec::framed_impl: frame decoded from buffer
2024-07-23T22:57:42.7997950Z 2024-07-23T22:57:42.783+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:42.7999643Z 2024-07-23T22:57:42.783+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:42.8001646Z 2024-07-23T22:57:42.783+0000 [TRACE] tokio_util::codec::framed_impl: frame decoded from buffer
2024-07-23T22:57:42.8003331Z 2024-07-23T22:57:42.783+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:42.8149136Z 2024-07-23T22:57:42.813+0000 [DEBUG] turborepo_repository::discovery: discovering packages using caching strategy
2024-07-23T22:57:42.8151446Z 2024-07-23T22:57:42.813+0000 [DEBUG] turborepo_lib::run::task_access: TASK ACCESS TRACE: unable to restore config from cache
2024-07-23T22:57:42.8163122Z • Packages in scope: dram
2024-07-23T22:57:42.8164407Z 2024-07-23T22:57:42.816+0000 [TRACE] turborepo_lib::task_hash: scm running in git mode
2024-07-23T22:57:42.8165504Z • Running test in 1 packages
2024-07-23T22:57:42.8166122Z • Remote caching disabled
2024-07-23T22:57:42.8167867Z 2024-07-23T22:57:42.816+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8170291Z 2024-07-23T22:57:42.816+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8172839Z 2024-07-23T22:57:42.816+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8177327Z 2024-07-23T22:57:42.816+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8179764Z 2024-07-23T22:57:42.816+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8184584Z 2024-07-23T22:57:42.816+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8190590Z 2024-07-23T22:57:42.818+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8192785Z 2024-07-23T22:57:42.818+0000 [DEBUG] turborepo_telemetry: Starting telemetry event queue flush (num_events=10)
2024-07-23T22:57:42.8194485Z 2024-07-23T22:57:42.818+0000 [DEBUG] turborepo_telemetry: Done telemetry event queue flush
2024-07-23T22:57:42.8201417Z 2024-07-23T22:57:42.819+0000 [TRACE] log: signal: Want
2024-07-23T22:57:42.8205563Z 2024-07-23T22:57:42.817+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8208150Z 2024-07-23T22:57:42.820+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8211046Z 2024-07-23T22:57:42.820+0000 [TRACE] log: signal: Want
2024-07-23T22:57:42.8212916Z 2024-07-23T22:57:42.820+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8322066Z 2024-07-23T22:57:42.832+0000 [DEBUG] turborepo_lib::run::global_hash: global hash env vars ["CI", "EDGEDB_BRANCH", "EDGEDB_INSTANCE", "EDGEDB_SECRET_KEY"]
2024-07-23T22:57:42.8339291Z 2024-07-23T22:57:42.833+0000 [DEBUG] globwalk: processing includes: [".editorconfig", ".eslintrc.cjs", ".prettierignore", "prettier.config.js", "tsconfig.json", "tsconfig.typecheck.json"]
2024-07-23T22:57:42.8342136Z 2024-07-23T22:57:42.833+0000 [DEBUG] globwalk: processing excludes: ["**/node_modules/**"]
2024-07-23T22:57:42.8344520Z 2024-07-23T22:57:42.833+0000 [DEBUG] globwalk: '/home/runner/work/apps/apps/prettier.config.js' doesn't have metadata
2024-07-23T22:57:42.8349144Z 2024-07-23T22:57:42.833+0000 [DEBUG] globwalk: processed includes: ["/home/runner/work/apps/apps/.editorconfig", "/home/runner/work/apps/apps/.eslintrc.cjs", "/home/runner/work/apps/apps/.prettierignore", "/home/runner/work/apps/apps/prettier.config.js", "/home/runner/work/apps/apps/tsconfig.json", "/home/runner/work/apps/apps/tsconfig.typecheck.json"]
2024-07-23T22:57:42.8351962Z 2024-07-23T22:57:42.833+0000 [DEBUG] globwalk: processed excludes: ["/home/runner/work/apps/apps/**/node_modules/**"]
2024-07-23T22:57:42.8407361Z 2024-07-23T22:57:42.840+0000 [DEBUG] turborepo_lib::run::global_hash: external deps hash: c8d5aac9959edec9
2024-07-23T22:57:42.8408971Z 2024-07-23T22:57:42.840+0000 [DEBUG] turborepo_lib::run: running visitor
2024-07-23T22:57:42.8410899Z 2024-07-23T22:57:42.840+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8412941Z 2024-07-23T22:57:42.840+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8414351Z 2024-07-23T22:57:42.840+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8415693Z 2024-07-23T22:57:42.840+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8416814Z 2024-07-23T22:57:42.840+0000 [DEBUG] turborepo_lib::task_hash: auto detected framework for dram
2024-07-23T22:57:42.8417805Z 2024-07-23T22:57:42.840+0000 [DEBUG] turborepo_lib::task_hash: framework: vite, env_prefix: ["VITE_*"]
2024-07-23T22:57:42.8422075Z 2024-07-23T22:57:42.842+0000 [DEBUG] turborepo_lib::task_hash: task hash env vars for dram:codegen
2024-07-23T22:57:42.8423053Z  vars: []
2024-07-23T22:57:42.8424373Z 2024-07-23T22:57:42.842+0000 [DEBUG] turborepo_lib::task_graph::visitor: task dram#codegen hash is a29164775e0e0a4c
2024-07-23T22:57:42.8426589Z 2024-07-23T22:57:42.842+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8429058Z 2024-07-23T22:57:42.842+0000 [DEBUG] turborepo_telemetry::config: Telemetry config path: /home/runner/.config/turborepo/telemetry.json
2024-07-23T22:57:42.8430816Z 2024-07-23T22:57:42.842+0000 [TRACE] turborepo_lib::process: acquiring lock for spawning (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:42.8432886Z 2024-07-23T22:57:42.842+0000 [TRACE] turborepo_lib::process: acquired lock for spawning (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:42.8437072Z 2024-07-23T22:57:42.843+0000 [TRACE] log: registering event source with poller: token=Token(140667585312384), interests=READABLE | WRITABLE
2024-07-23T22:57:42.8439190Z 2024-07-23T22:57:42.843+0000 [TRACE] log: registering event source with poller: token=Token(140667585312896), interests=READABLE | WRITABLE
2024-07-23T22:57:42.8441618Z 2024-07-23T22:57:42.843+0000 [TRACE] log: registering event source with poller: token=Token(140667585313280), interests=READABLE | WRITABLE
2024-07-23T22:57:42.8443321Z 2024-07-23T22:57:42.843+0000 [TRACE] turborepo_lib::process: releasing lock for spawning (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:42.8445288Z 2024-07-23T22:57:42.843+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:42.8446651Z 2024-07-23T22:57:42.843+0000 [DEBUG] turborepo_lib::process::child: waiting for task
2024-07-23T22:57:42.8804068Z 2024-07-23T22:57:42.880+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:42.8805922Z 2024-07-23T22:57:42.880+0000 [TRACE] tokio_util::codec::framed_impl: frame decoded from buffer
2024-07-23T22:57:42.8807733Z 2024-07-23T22:57:42.880+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:42.8809426Z 2024-07-23T22:57:42.880+0000 [TRACE] tokio_util::codec::framed_impl: frame decoded from buffer
2024-07-23T22:57:42.8810718Z 2024-07-23T22:57:42.880+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:42.9080176Z 2024-07-23T22:57:42.907+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:42.9082567Z 2024-07-23T22:57:42.907+0000 [TRACE] tokio_util::codec::framed_impl: frame decoded from buffer
2024-07-23T22:57:42.9084693Z 2024-07-23T22:57:42.907+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:42.9086729Z 2024-07-23T22:57:42.907+0000 [TRACE] tokio_util::codec::framed_impl: frame decoded from buffer
2024-07-23T22:57:42.9087965Z 2024-07-23T22:57:42.907+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:42.9810043Z 2024-07-23T22:57:42.980+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:42.9812067Z 2024-07-23T22:57:42.980+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:42.9815009Z 2024-07-23T22:57:42.980+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:42.9817557Z 2024-07-23T22:57:42.980+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:42.9819373Z 2024-07-23T22:57:42.980+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:42.9822329Z 2024-07-23T22:57:42.980+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:43.2092096Z 2024-07-23T22:57:43.208+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:43.2094002Z 2024-07-23T22:57:43.208+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:43.2108436Z 2024-07-23T22:57:43.210+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:43.2110423Z 2024-07-23T22:57:43.210+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:43.8417870Z 2024-07-23T22:57:43.841+0000 [DEBUG] turborepo_telemetry: Starting telemetry event queue flush (num_events=7)
2024-07-23T22:57:43.8419956Z 2024-07-23T22:57:43.841+0000 [DEBUG] turborepo_telemetry: Done telemetry event queue flush
2024-07-23T22:57:43.8420928Z 2024-07-23T22:57:43.841+0000 [TRACE] log: signal: Want
2024-07-23T22:57:43.8421653Z 2024-07-23T22:57:43.841+0000 [TRACE] log: signal: Want
2024-07-23T22:57:43.8472916Z 2024-07-23T22:57:43.847+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:43.8474161Z 2024-07-23T22:57:43.847+0000 [TRACE] tokio_util::codec::framed_impl: frame decoded from buffer
2024-07-23T22:57:43.8475260Z 2024-07-23T22:57:43.847+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:43.8985552Z 2024-07-23T22:57:43.898+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:43.8987009Z 2024-07-23T22:57:43.898+0000 [TRACE] tokio_util::codec::framed_impl: frame decoded from buffer
2024-07-23T22:57:43.8988204Z 2024-07-23T22:57:43.898+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:43.8989246Z 2024-07-23T22:57:43.898+0000 [TRACE] tokio_util::codec::framed_impl: frame decoded from buffer
2024-07-23T22:57:43.8990285Z 2024-07-23T22:57:43.898+0000 [TRACE] tokio_util::codec::framed_impl: attempting to decode a frame
2024-07-23T22:57:45.2814924Z 2024-07-23T22:57:45.281+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.2817935Z 2024-07-23T22:57:45.281+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:45.2840925Z 2024-07-23T22:57:45.283+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.2843958Z 2024-07-23T22:57:45.283+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:45.2909883Z 2024-07-23T22:57:45.290+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.2913115Z 2024-07-23T22:57:45.290+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:45.2979814Z 2024-07-23T22:57:45.297+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.2981431Z 2024-07-23T22:57:45.297+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:45.3037744Z 2024-07-23T22:57:45.303+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.3040973Z 2024-07-23T22:57:45.303+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:45.3318071Z 2024-07-23T22:57:45.331+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.3319682Z 2024-07-23T22:57:45.331+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:45.3572147Z 2024-07-23T22:57:45.356+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.3575759Z 2024-07-23T22:57:45.356+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:45.3578516Z 2024-07-23T22:57:45.357+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.3581940Z 2024-07-23T22:57:45.357+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:45.3584638Z 2024-07-23T22:57:45.357+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.3587542Z 2024-07-23T22:57:45.357+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:45.4094379Z 2024-07-23T22:57:45.409+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.4097391Z 2024-07-23T22:57:45.409+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:45.4100119Z 2024-07-23T22:57:45.409+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.4103025Z 2024-07-23T22:57:45.409+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:45.4105764Z 2024-07-23T22:57:45.410+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.4107353Z 2024-07-23T22:57:45.410+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T22:57:45.4108726Z 2024-07-23T22:57:45.410+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.4109649Z 2024-07-23T22:57:45.410+0000 [TRACE] turborepo_lib::process::child: processing stdout line
2024-07-23T22:57:45.4111143Z 2024-07-23T22:57:45.410+0000 [TRACE] turborepo_lib::process::child: watching exit channel of (/home/runner/work/apps/apps/apps/dram) /opt/hostedtoolcache/node/20.15.1/x64/bin/npm run codegen
2024-07-23T23:00:42.7822192Z 2024-07-23T23:00:42.781+0000 [TRACE] log: signal: Closed
2024-07-23T23:00:42.7823682Z 2024-07-23T23:00:42.781+0000 [DEBUG] log: Sending warning alert CloseNotify
2024-07-23T23:00:42.7825159Z 2024-07-23T23:00:42.782+0000 [TRACE] log: deregistering event source from poller
2024-07-23T23:08:14.9324400Z ##[error]The operation was canceled.
```

--- a/crates/turborepo-lib/src/task_graph/visitor.rs
+++ b/crates/turborepo-lib/src/task_graph/visitor.rs
@@ -952,6 +952,13 @@ impl ExecContext {
             }
         }
 
+        // Even if user does not have the TUI and cannot interact with a task, we keep
+        // stdin open for persistent tasks as some programs will shut down if stdin is
+        // closed.
+        if !self.takes_input {
+            process.stdin();
+        }
+
         let mut stdout_writer = self
             .task_cache
             .output_writer(prefixed_ui.task_writer())

